app/Controllers/PostControllers.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+
+use App\View;
+use App\Models\Post;
+use App\Models\User;
+use App\Models\Comment;
+
+class PostControllers
+{
+    CONST REQUEST_METHOD = 'POST';
+    protected Post $postModel;
+    protected User $userModel;
+    protected Comment $commentModel;
+
+
+
+    public function __construct()
+    {
+        $this->postModel = new Post();
+        $this->userModel = new User();
+        $this->commentModel = new Comment();
+    }
+
+
+    /**
+     * function to get All Blog Post Data.
```

Comment:
Don't need to comment 'function to' or 'function for' before the function comments.